hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/contract/ContractCallTransitionLogic.java

```diff
@@ -230,9 +230,13 @@ private ResponseCodeEnum validateSemantics(final TransactionBody transactionBody
         if (possiblySanityCheckOp(op, target)) {
             try {
                 final var receiver = accountStore.loadContract(target.toId());
-                validateTrue(receiver != null && receiver.isSmartContract(), INVALID_CONTRACT_ID);
+                if (!receiver.isSmartContract()) {
+                    return INVALID_CONTRACT_ID;
+                }
             } catch (InvalidTransactionException e) {
-                return e.getResponseCode();
+                // Continue silently as we wish to allow calls to deleted and non-existent contracts
```

Comment:
Is this meant to be in this mod-service PR?

Reply:
No this should be removed and any modifications to this class should be with the parent.